pymc/distributions/timeseries.py

```diff
@@ -33,6 +38,193 @@
 ]
 
 
+class GaussianRandomWalkRV(RandomVariable):
+    """
+    GaussianRandomWalk Random Variable
+    """
+
+    name = "GaussianRandomWalk"
+    ndim_supp = 1
+    ndims_params = [0, 0, 0, 0]
+    dtype = "floatX"
+    _print_name = ("GaussianRandomWalk", "\\operatorname{GaussianRandomWalk}")
+
+    def make_node(self, rng, size, dtype, mu, sigma, init, steps):
```

Comment:
This method should return super().make_node(...). Not sure what happened with my changes xD. Must have changed and not retested locally

Here is a reference distribution doing the correct thing: https://github.com/pymc-devs/pymc/blob/741d45540016ab52a15ad5bff521614b6ae1d835/pymc/distributions/multivariate.py#L291

Reply:
Ok great, ill take a look and as always Ill have questions, in the spirit of learning :)

Reply:
Thanks buddy! That did the trick, and I really appreciate you showing me what to do